tests/test_parametric_components/test_PortCutterRectangular.py

```diff
@@ -3,19 +3,55 @@
 
 import paramak
 
+import pytest
+
 
 class TestPortCutterRectangular(unittest.TestCase):
 
-    def test_creation(self):
-        """Checks a PortCutterRectangular creation."""
-
-        test_component = paramak.PortCutterRectangular(
-            distance=3,
-            z_pos=0,
-            height=0.2,
-            width=0.4,
-            fillet_radius=0.02,
-            azimuth_placement_angle=[0, 45, 90, 180]
+    def setUp(self):
+        self.test_shape = paramak.PortCutterRectangular(
+            width=40, height=40, distance=300
         )
 
-        assert test_component.solid is not None
+    def test_default_parameters(self):
+        """Checks that the default parameters of a PortCutterRectangular are correct."""
+
+        assert self.test_shape.center_point == (0, 0)
+        assert self.test_shape.workplane == "ZY"
+        assert self.test_shape.rotation_axis == "Z"
+        assert self.test_shape.extrusion_start_offset == 1
+        assert self.test_shape.fillet_radius is None
+        assert self.test_shape.stp_filename == "PortCutterRectangular.stp"
+        assert self.test_shape.stl_filename == "PortCutterRectangular.stl"
+        assert self.test_shape.name == "rectangular_port_cutter"
+        assert self.test_shape.material_tag == "rectangular_port_cutter_mat"
```

Comment:
As said a bit earlier, these tests may fail if another test which changes the attributes is run before this one. Perhaps instead of using `self.test_shape` a new shape could be created ? or these assertions could be moved to setUp ?

Reply:
See above